frontend/src/pages/Deckbuilder.tsx

```diff
@@ -124,12 +128,12 @@ const DeckNameInput = styled.input`
   };
 `;
 
-const SaveDeckButton = styled.button`
+const SaveDeckButton = styled.button<{isSaving:boolean}>`
```

Comment:
Here another thing i would like to know about it;)

Reply:
I just wanted the button to be grayed out and disabled until the page reloads.
But thanks for asking, I forgot to reset the state there :D